front/app/containers/Admin/projects/components/ProjectRow/index.tsx

```diff
@@ -50,24 +52,32 @@ export interface Props {
   actions: ButtonAction[];
   hidePublicationStatusLabel?: boolean;
   className?: string;
-  showMoreActions?: boolean;
+  hideMoreActions?: boolean;
 }
 
 const ProjectRow = ({
   publication,
   actions,
   hidePublicationStatusLabel,
   className,
-  showMoreActions = false,
+  hideMoreActions = false,
 }: Props) => {
-  const [isBeingDeleted, _setIsBeingDeleted] = useState<boolean>(false);
   const [error, setError] = useState<string | null>(null);
   const authUser = useAuthUser();
   const projectId = publication.publicationId;
+  const project = useProject({ projectId });
   const publicationStatus = publication.attributes.publication_status;
+  const folderId = project?.attributes.folder_id;
+
+  if (isNilOrError(authUser)) {
+    return null;
+  }
   const userCanModerateProject =
-    !isNilOrError(authUser) &&
-    canModerateProject(publication.publicationId, { data: authUser });
+    // This means project is in a folder
+    (typeof folderId === 'string' && userModeratesFolder(authUser, folderId)) ||
```

Comment:
This was missing. If user is moderator of folder that contains project, they are project moderators as well. This is something we need to change in the entire app in fact. Bringing it up with the front-end team.

Reply:
Without being 100% certain, I think it is: canModerateProject checks for the project_id as part of the project_moderator role a user can have. A folder moderator doesn't necessarily have that. The code is confusing at the moment.

Reply:
(Our tests don't cover this case/folder moderators.)